includes/Promotions.js

```diff
@@ -0,0 +1,42 @@
+var node = document.getElementById("custom_tab_data");
+node.innerHTML = `<div style="width: 100%;height:350px">
+    <p style=" text-align: center;padding-top:40px;line-height: 20px; font-size: 16px; font-weight: 600;">Boost Your Online Store Sales</p>
+    <p style=" text-align: center;line-height: 20px; font-size: 13px;">Maximize your sales by creating effective </br> promotions and campaigns like:</p>
+    <div style="text-align: center;">
+      <div style="display: flex;flex:1; align-items: center; justify-content:  space-around ;margin:0 auto; width: 50%;padding:20px 0">
+        <div style=" text-align: center;">
```

Comment:
@manikantakailasa can we move svg files under `src/icons` directory & import here?

Reply:
with src path it is not able to find the files getting 404 error and import wont work in this file

Reply:
path is correct not able to load it correctly